internal/db/rdb/mysql/mysql.go

```diff
@@ -168,8 +178,17 @@ func (m *mySQLClient) Ping(ctx context.Context) (err error) {
 }
 
 // Close closes the connection of MySQL database.
-// If the connection is already closed or closing conncection is failed, it returns error.
+// If the connection is already closed or closing connection is failed, it returns error.
 func (m *mySQLClient) Close(ctx context.Context) (err error) {
+	if m.session == nil {
+		err = errors.ErrMySQLSessionNil
+		log.Errorf(
+			"err: %s, { db: %s, host: %s, port: %d, user: %s, name: %s, charset: %s } ",
+			err.Error(), m.db, m.host, m.port, m.user, m.name, m.charset,
+		)
```

Comment:
IMO:
Since the same code appears multiple times, I thought it would be cleaner if I created a new function and formatted it and output an error inside it.
if I made a mistake, I am sorry.

Reply:
@hlts2 e. g. below code?
```go
func (m *mySQLClient) outputNilSessionLog(err error) {
	log.Errorf(
		"err: %s, { db: %s, host: %s, port: %d, user: %s, name: %s, charset: %s } ",
		err.Error(), m.db, m.host, m.port, m.user, m.name, m.charset,
	)}
}
```

Reply:
yes, It looks good to me!